apps/web/src/App.tsx

```diff
@@ -58,7 +58,7 @@ if (SENTRY_DSN) {
     beforeSend(event: Sentry.Event) {
       const logRocketSession = LogRocket.sessionURL;
 
-      if (logRocketSession !== null || event !== '' || event !== undefined) {
+      if (logRocketSession !== null || (event as string) !== '' || event !== undefined) {
```

Comment:
Build stage was failing for uncompatibility between string and Sentry.Event. Forcing the string check.
Might worth to review this condition and turn it upside down.